static/dashboard.html

```diff
@@ -49,6 +55,11 @@ <h3>Table reservation chart</h3>
 
     function createRemindButton(bookingId) {
       var button = document.createElement("button");
+      if (bookingId == undefined){
+        button.className = "btn btn-primary";
```

Comment:
If we make button filled with white, that should improve visibility of which table is booked. What do you think?

Reply:
It works, come on!

Reply:
😃